migrations/item-migrations.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+const { Item }  = require('../database/models')
+
+module.exports = {
+  up: ( migration, DataTypes ) => {
+    return migration.createTable( Item.tableName, Item.attributes )
+  },
```

Comment:
omit braces and `return` statement for single line functions (this applies elsewhere in this commit)